meson.build

```diff
@@ -942,6 +942,12 @@ if features['libplacebo']
     endif
 endif
 
+dovi = dependency('dovi', required: false)
```

Comment:
I think generally there is option added for each feature. `required: get_option('dovi')` or something should suffice, and entry in `meson_options.txt`

Reply:
also, probably version check for `libdovi` would be useful?

Reply:
> also, probably version check for `libdovi` would be useful?

The library isn't really distributed anywhere, and interfaces used here have been there forever.
So I don't think a minimum is necessary.